src/Catalog/Helpers/FeedHelpers.cs

```diff
@@ -41,15 +41,22 @@ public class PackageDetails
             public string LicenseReportUrl { get; set; }
         }
 
-        public static HttpClient CreateHttpClient(bool verbose)
+        /// <summary>
+        /// Creates an HttpClient for reading the feed.
+        /// </summary>
+        public static HttpClient CreateHttpClient(Func<HttpMessageHandler> handlerFunc)
         {
-            var handlerFunc = CommandHelpers.GetHttpMessageHandlerFactory(verbose);
-
             var handler = (handlerFunc != null) ? handlerFunc() : new WebRequestHandler { AllowPipelining = true };
-
             return new HttpClient(handler);
         }
 
+        /// <summary>
+        /// Reads and returns metadata from the top level of the catalog index.json.
+        /// </summary>
+        /// <param name="propertyName">
+        /// Metadata field to return.
+        /// Possible values include "nuget:lastCreated", "nuget:lastDeleted", and "nuget:lastEdited", which are the timestamps of the cursor of the catalog.
```

Comment:
This is existing code (that I added a comment to). I agree that it's ugly but I think fixing it is out of the scope of this PR.